index.js

```diff
@@ -18,30 +19,48 @@ var startTime = function() {
   });
 };
 
+var potentiallyMigrateDatabase = function() {
+  return new Promise(function (resolve, reject) {
+    postgrator.setConfig({
+      migrationDirectory: __dirname + '/migrations',
+      driver: 'pg',
+      connectionString: process.env.POSTGRESQL_URL
+    });
+
+    postgrator.migrate('201505271423', function(err, migrations) {
+      if (err) {
+        reject(err);
+      } else {
+        postgrator.endConnection(function() {
+          resolve(migrations);
+        });
+      }
+    });
+  });
+};
+
 var loop = function () {
   var starttime;
   startTime()
     .then(function (time) {
       starttime = time;
     })
-
+    .then(potentiallyMigrateDatabase)
+    .then(function(migrations) {
+      console.log('Sucessfully migrated', migrations);
+    })
     .then(couch2pg)
     .then(function () {
       console.log('Imported successfully at ' + Date());
-    }, function (err) {
-      console.log('Import errored at ' + Date());
-      console.log(err);
     })
-
     .then(xmlforms)
     .then(function () {
       console.log('XML forms completed at ' + Date());
-    }, function (err) {
-      console.log('XML forms errored at ' + Date());
-      console.log(err);
     })
-
-    .then(function () {
+    .catch(function(err) {
+      console.error(err);
```

Comment:
This changed slightly in the next commit, but I'm still not entirely sure if the error catching works the same as before, so I'll leave the comment here.

Similarly, I think we should add a timestamp onto the equivalent to this line in the next commit.